frontend/src/app/components/containers/complaints/outcomes/ceeb/ceeb-decision/decision-form.tsx

```diff
@@ -325,32 +278,11 @@ export const DecisionForm: FC<props> = ({
     };
 
     const _isActionValid = (data: Decision, _isValid: boolean): boolean => {
-      if (data.actionTaken && (!data.actionTakenDate || !data.assignedTo)) {
+      if (data.actionTaken && !data.actionTakenDate) {
         if (!data.actionTakenDate) {
```

Comment:
We should be able to get rid of the second `if` here since the outside `if` already confirms `!data.actionTakenDate` (before it didn't).